Makefile

```diff
@@ -28,33 +28,38 @@ test-all: setup-chain
 	@echo "Waiting for chain to resume..."
 	@sleep 7
 
-	TEST_TYPE=multi-msg-load bash ./scripts/tests/multi_msg_load.sh -n 50
-	TEST_TYPE=query-load bash ./scripts/tests/query_load.sh -n 50
-	TEST_TYPE=send-load bash ./scripts/tests/send_load.sh -n 50
-	TEST_TYPE=single-msg-load bash ./scripts/tests/single_msg_load.sh -n 50
+	TEST_TYPE=multi-msg-load bash ./scripts/tests/multi_msg_load.sh
```

Comment:
@akhilkumarpilli can you update README to reflect these changes i.e., setting `NUM_TXS` env